### DISCUSSION TITLE
Pull Requests and Issues templates

### DISCUSSION THREAD

**Discussion by tutods** [nest-modules/mailer#819] [General]
Hello everyone.

What you think about create Pull Requests and Issues templates to turn easy this process to the community.
For example:
- 2 issues templates:
   -  BUGS: with a specific structure with reproduce steps, etc..
   - FEATURE: for new feature
   - SUGGESTION: for suggestions

And a default structure for new pull requests.

Let me know what you think! 😉

**Comment by juandav** [comment]
I think it's great, we could create a more suitable structure for the new inclusions, we must also remember that the commits to be made are under a convention.
